src/common/options/global.yaml.in

```diff
@@ -5903,12 +5903,12 @@ options:
   level: dev
   default: 0
   with_legacy: true
-- name: filestore_queue_low_threshhold
+- name: filestore_queue_low_threshold
```

Comment:
I think changing option names is out of scope for this PR.  And as above, Filestore is deprecated anyway, so I suggest not going through all the hassle.  Same with the below three changes.

Reply:
+1, would avoid changing user-visible pieces like option names that may be part of config files/scripts/generally the UI surface area